### PR TITLE
Add specs

### LINKED ISSUE #30: Implement specs
It looks like this linter does not currently have any specs written for it, adding it to the list being maintained [here](https://github.com/AtomLinter/Meta/issues/11) for tracking.

Specs do not need to be super detailed, as they should mainly test whether the package is properly running the linter plugin and parsing its messages. Leave testing of the linter functionality to the author of the linter.

If you need any help feel free to ask!

### PR DESCRIPTION
**Pull request by 1000ch**
Will fix #30.
@Arcanemagus @steelbrain Would you configure CI for this?

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import { Range } from 'atom';
+import * as jsonlint from 'jsonlint';
+
+export default class LinterJsonLint {
+
+  static regex = '.+?line\\s(\\d+)';
+
+  static activate() {
+    require('atom-package-deps').install('linter-jsonlint');
```
Comment (steelbrain) [inline lib/index.js]: You can drop this parameter, it's completely optional. But you can also keep it if you want, fine either way

Comment (1000ch) [inline lib/index.js]: Now I can see. I will drop it soon.

**Comment by steelbrain** [comment]
@1000ch Done

**Comment by 1000ch** [comment]
@steelbrain :beer:

**Comment by Arcanemagus** [comment]
2 down :tada:, way to go!

**Pull request merged**